sonic_data_client/mixed_db_client.go

```diff
@@ -478,8 +486,20 @@ func (c *MixedDbClient) tableData2Msi(tblPath *tablePath, useKey bool, op *strin
 	var err error
 	var fv map[string]string
 
-	//Only table name provided
-	if tblPath.tableKey == "" {
+	if tblPath.tableName == "" {
+		// Did no provide table name
+		// Get all tables in the DB
+		if tblPath.dbName == "COUNTERS_DB" {
+			return fmt.Errorf("Can not read all tables in COUNTERS_DB")
```

Comment:
>Can not [](http://example.com/codeflow?start=22&length=7)

This is interesting limitation. Is there a reason? Is it in any design doc? Or just not implemented today.

Reply:
tables in COUNTERS_DB other than COUNTERS table doesn't have keys.
So, I prefer not to implement it today.

Reply:
I have Added TODO and new unit test.